host_core/native/hostcore_wasmcloud_native/src/wasmruntime.rs

```diff
@@ -42,12 +53,12 @@ pub struct ElixirHandler {
 }
 
 #[async_trait]
-impl Handle<HostInvocation> for ElixirHandler {
+impl Handle<Invocation> for ElixirHandler {
```

Comment:
Please do not use the unqualified form of `Invocation` here. There is an invocation struct already inside this NIF that is used for over-the-wire RPCs (in inv.rs). If libwasmcloud uses this type name, then perhaps create a type alias for it like `WasmcloudInvocation` or `WebAssemblyInvocation`, something to differentiate this invocation from the network/RPC invocation